release/cloud-build-configs/kfserving-controller.cloud-build.yaml

```diff
@@ -1,7 +0,0 @@
-steps:
-- name: 'gcr.io/cloud-builders/go'
-  args: ['install', './cmd/manager']
-  env:  ['PROJECT_ROOT=github.com/kubeflow/kfserving']
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/kfserving/kfserving-controller:latest', '.']
-images: ['gcr.io/kfserving/kfserving-controller:latest']
```

Comment:
So this is no longer needed, is it configured here(https://console.cloud.google.com/cloud-build/triggers?project=kfserving&angularJsUrl=%2Fcloud-build%2Ftriggers%3Fproject%3Dkfserving&authuser=1) now ?

Reply:
Yeah. I managed to configure them directly off the Docker files ;)